### DISCUSSION TITLE
How to integrate plots for input data?

### DISCUSSION THREAD

**Discussion by Giuseppe-Accardo** [oegedijk/explainerdashboard#247] [Q&A]
Congratulations for the fantastic library! 

I wanted to ask you if is it possible to include a tab or section for input data used? For example, show the statistics of the input data or the distribution like this:
![image](https://user-images.githubusercontent.com/33551525/211363728-c0c3a8e2-6091-4589-9af1-d5af3b629f45.png)

Thank you!

**Comment by oegedijk** [comment]
@Giuseppe-Accardo I agree this would be a great additional component. Don't have time for it myself right now, but would be happy to accept a PR!
